docs/features/techdocs/addons--new.md

```diff
@@ -0,0 +1,204 @@
+| [`ReportIssueAddonExtension`](https://backstage.io/docs/reference/plugin-techdocs-module-addons-contrib.reportissue)                   | `@backstage/plugin-techdocs-module-addons-contrib/alpha` | Allows TechDocs users to select a portion of text on a TechDocs page and open an issue against the repository that contains the documentation, populating the issue description with the selected text according to a configurable template.                                                                                                                        |
+| [`TextSizeAddonExtension`](https://backstage.io/docs/reference/plugin-techdocs-module-addons-contrib.textsize)                         | `@backstage/plugin-techdocs-module-addons-contrib/alpha` | This TechDocs addon allows users to customize text size on documentation pages, they can select how much they want to increase or decrease the font size via slider or buttons. The default value for font size is 100% and this setting is kept in the browser's local storage whenever it is changed.                                                             |
+| [`LightBoxAddonExtension`](https://backstage.io/docs/reference/plugin-techdocs-module-addons-contrib.lightbox)                         | `@backstage/plugin-techdocs-module-addons-contrib/alpha` | This TechDocs addon allows users to open images in a light-box on documentation pages, they can navigate between images if there are several on one page. The image size of the light-box image is the same as the image size on the document page. When clicking on the zoom icon it zooms the image to fit in the screen (similar to `background-size: contain`). |
+
+Got an Addon to contribute? Feel free to add a row above!
+
+## Creating an Addon
+
+The simplest Addons are plain old react components that get rendered in
+specific locations within a TechDocs site. To package such a react component as
+an Addon, follow these steps:
+
+1. Write the component in your plugin like any other component
+2. Create the addon extension using the `TechDocsAddonBlueprint` and export the addon extension from your plugin
+
+```ts
+// plugins/your-plugin/src/plugin.ts
+
+import { TechDocsAddonLocations } from '@backstage/plugin-techdocs-react';
+import { TechDocsAddonBlueprint } from '@backstage/plugin-techdocs-react/alpha';
+import { CatGifComponent } from './addons';
+
+// ...
+
+export const catGifAddonExtension = TechDocsAddonBlueprint.make({
+  name: 'CatGif',
```

Comment:
Stick to the naming patterns, event, or maybe especially for docs 😁  https://backstage.io/docs/frontend-system/architecture/naming-patterns#extensions